src/redux/reducers/genes/getInitialState.js

```diff
@@ -13,12 +13,20 @@ const getInitialState = () => ({
     data: {},
   },
   expression: {
-    loading: [],
-    error: false,
-    matrix: new ExpressionMatrix(),
-    downsampledMatrix: new ExpressionMatrix(),
-    downsampledCellOrder: [],
     views: {},
+    full: {
+      loading: [],
+      error: false,
+      matrix: new ExpressionMatrix(),
+    },
+    downsampled: {
```

Comment:
Fully split up downsampled and full expression in redux (it was already halfway there) so that it is easier to manage them and there's no chance of one loading affecting the other one.